analysis/src/CompletionFrontEnd.ml

```diff
@@ -804,8 +957,16 @@ let completionWithParser1 ~currentFile ~debug ~offset ~path ~posCursor ~text =
                 ~firstCharBeforeCursorNoWhite
             | None -> None
           in
-
-          setResultOpt argCompletable
+          match argCompletable with
+          | None -> (
+            match exprToContextPath funExpr with
```

Comment:
Same match a few lines above.
Also, this case looks nearly identical to the one above for `"|."`.